vamores/vamores.md

```diff
@@ -0,0 +1,9 @@
+vamores Vincent Amores
+hotfix/no-ref
+bugfix/core-21
+feature/wtd-22
```

Comment:
this is not right - needs the description at the end.

Reply:
@mldasco  / @gilbertyin, added description.